docker/config/requirements.txt

```diff
@@ -72,7 +72,7 @@ natsort==7.1.1
 numpy==1.20.2
 openapi-schema-validator==0.1.5
 openapi-spec-validator==0.3.0
-pandas==1.2.4
```

Comment:
We should put this in alan-jobs/dags/requirements.txt

Reply:
this was already here, so should I then remove it from the MWAA repo as we can't have 2 requirements with contradictory versions ?

Reply:
From our sync: modifying in our repo works for the prod environment but fails for local install. I'll find a workarround for the local install.